code-scanning/properties/ethicalcheck.properties.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "EthicalCheck",
+  "creator": "APIsec",
+  "description": "EthicalCheck addresses the critical need to continuously security test APIs in development and in production. EthicalCheck provides the industry’s only free & automated API security testing service that uncovers security vulnerabilities using OWASP API list. Developers relies on EthicalCheck to evaluate every update and release, ensuring that no APIs go to production with exploitable vulnerabilities. You develop the application and API, we bring complete and continuous security testing to you, accelerating development.Know your API and Applications are secure with EthicalCheck – our free & automated API security testing service.",
```

Comment:
This description is too big. Can we make it smaller to fit in 3 lines here:
![image](https://user-images.githubusercontent.com/25999504/196860647-1be9eebe-83c3-4c5c-8aab-3af3f29eb149.png)

Reply:
Created pull request for same. https://github.com/actions/starter-workflows/pull/1787